libs/pageserver_api/src/models.rs

```diff
@@ -486,11 +486,11 @@ pub struct EvictionPolicyLayerAccessThreshold {
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
 pub struct ThrottleConfig {
     pub task_kinds: Vec<String>, // TaskKind
-    pub initial: usize,
+    pub initial: u32,
     #[serde(with = "humantime_serde")]
     pub refill_interval: Duration,
-    pub refill_amount: NonZeroUsize,
-    pub max: usize,
+    pub refill_amount: NonZeroU32,
+    pub max: u32,
     pub fair: bool,
```

Comment:
This field is now ignored - the new implementation is always fair.

The field _is_ set in the production pageserver configs.

But we don't have deny_unknown_fields set (a blunder!) so I think we can remov the `fair` field in this PR, wait for it to roll out,t hen remove the `fair` field from the production configs.